packages/firebase_auth/firebase_auth/lib/src/firebase_auth.dart

```diff
@@ -647,6 +647,12 @@ class FirebaseAuth extends FirebasePluginPlatform {
     return _delegate.verifyPasswordResetCode(code);
   }
 
+  Future<UserCredentialPlatform> signInWithAuthProvider(
```

Comment:
All the methods have doc comments in this file. Is this intended that this method has not a doc comment?

Reply:
Added, thanks :)